From 22ac6d1c55e955171d94d0e4901684681f1d5c84 Mon Sep 17 00:00:00 2001
From: Tao Zeng <tao.zeng@amlogic.com>
Date: Mon, 27 Jul 2015 14:15:34 +0800
Subject: [PATCH 5728/5965] =?UTF-8?q?PD#110040:=20cec=EF=BC=9A=20fix=20boo?=
 =?UTF-8?q?tup=20problem=20caused=20by=20cec=20service?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

change output of cec_config sysfs to avoid system service died

Change-Id: Icfb271a581d71eaacb3178a1bc7b768609f6ee4e
---
 drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c b/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c
index 76f003e3248c..cffedbbac6ff 100644
--- a/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c
+++ b/drivers/amlogic/hdmi/hdmi_tx_2/hdmi_tx_main.c
@@ -515,7 +515,7 @@ static ssize_t store_cec(struct device * dev, struct device_attribute *attr, con
 static ssize_t show_cec_config(struct device * dev, struct device_attribute *attr, char * buf)
 {
     int pos=0;
-    pos+=snprintf(buf+pos, PAGE_SIZE, "P_AO_DEBUG_REG0:0x%x\r\n", aml_read_reg32(P_AO_DEBUG_REG0));
+    pos+=snprintf(buf+pos, PAGE_SIZE, "0x%x\n", aml_read_reg32(P_AO_DEBUG_REG0));
     return pos;
 }
 
-- 
2.19.0

